packages/mermaid/src/diagrams/sankey/parser/sankey.spec.js

```diff
@@ -0,0 +1,156 @@
+        "Gas imports"               ->      40.719   -> "Ngas"
+        "Gas reserves"              ->      82.233   -> "Ngas"
+        "Gas"                       ->      0.129    -> "Heating and cooling - commercial"
+        "Gas"                       ->      1.401    -> "Losses"
+        "Gas"                       ->      151.891  -> "Thermal generation"
+        "Gas"                       ->      2.096    -> "Agriculture"
+        "Gas"                       ->      48.58    -> "Industry"
+        "Geothermal"                ->      7.013    -> "Electricity grid"
+        "H2 conversion"             ->      20.897   -> "H2"
+        "H2 conversion"             ->      6.242    -> "Losses"
+        "H2"                        ->      20.897   -> "Road transport"
+        "Hydro"                     ->      6.995    -> "Electricity grid"
+        "Liquid"                    ->      121.066  -> "Industry"
+        "Liquid"                    ->      128.69   -> "International shipping"
+        "Liquid"                    ->      135.835  -> "Road transport"
+        "Liquid"                    ->      14.458   -> "Domestic aviation"
+        "Liquid"                    ->      206.267  -> "International aviation"
+        "Liquid"                    ->      3.64     -> "Agriculture"
+        "Liquid"                    ->      33.218   -> "National navigation"
+        "Liquid"                    ->      4.413    -> "Rail transport"
+        "Marine algae"              ->      4.375    -> "Bio-conversion"
+        "Ngas"                      ->      122.952  -> "Gas"
+        "Nuclear"                   ->      839.978  -> "Thermal generation"
+        "Oil imports"               ->      504.287  -> "Oil"
+        "Oil reserves"              ->      107.703  -> "Oil"
+        "Oil"                       ->      611.99   -> "Liquid"
+        "Other waste"               ->      56.587   -> "Solid"
+        "Other waste"               ->      77.81    -> "Bio-conversion"
+        "Pumped heat"               ->      193.026  -> "Heating and cooling - homes"
+        "Pumped heat"               ->      70.672   -> "Heating and cooling - commercial"
+        "Solar PV"                  ->      59.901   -> "Electricity grid"
+        "Solar Thermal"             ->      19.263   -> "Heating and cooling - homes"
+        "Solar"                     ->      19.263   -> "Solar Thermal"
+        "Solar"                     ->      59.901   -> "Solar PV"
+        "Solid"                     ->      0.882    -> "Agriculture"
+        "Solid"                     ->      400.12   -> "Thermal generation"
+        "Solid"                     ->      46.477   -> "Industry"
+        "Thermal generation"        ->      525.531  -> "Electricity grid"
+        "Thermal generation"        ->      787.129  -> "Losses"
+        "Thermal generation"        ->      79.329   -> "District heating"
```

Comment:
You can add some unicode symbols to the test too.

Reply:
As you're already supporting double quote strings, unicode should already be supported. 
Only a test case might be sufficient.

Reply:
Understood, I'll check that

Reply:
No need for that, since we've changed the syntax